python/src/HSICEstimatorTargetSensitivity_doc.i.in

```diff
@@ -0,0 +1,79 @@
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::HSICEstimatorTargetSensitivity
+"Implement a HSIC estimator for target analysis.
+
+Available constructors:
+    HSICEstimatorTargetSensitivity()
+
+    HSICEstimatorImplementation(covarianceList, X, Y, estimatorType, filterFunction)
```

Comment:
no need to list ctors if there's only one

Reply:
Done